tags:
  {'tag_0': {size:  0.24, x: -0.7, y: 2.7, z: 0.9, qw: 1, qx: 1.57, qy: 0, qz: 1.57},
   'tag_1': {size:  0.24, x: -2.5, y: 2.7, z: 0.9, qw: 1, qx: 1.57, qy: 0, qz: 1.57},
   'tag_2': {size:  0.24, x: 1.5, y: 0.14, z: 1.2, qw: 1, qx: 0, qy: -1.5708, qz: -1.5708},
   'tag_3': {size:  0.24, x: 0.2, y: 2.7, z: 1.35, qw: 1, qx: 1.57, qy: 0, qz: 1.57},
   'tag_4': {size:  0.24, x: -0.17, y: -2.25, z: 0.9, qx: 1.5708 , qy: 3.141592, qz: 4.712},
   'tag_5': {size:  0.24, x: -2.86, y: -0.031721, z: 0.2, qw: 1, qx: -1.57, qy: 1.57, qz: 0},
   'tag_6': {size:  0.24, x: -3.4, y: 2.0, z: 1.2, qw: 1, qx: -1.5708, qy: 1.5708, qz: 0},
   'tag_7': {size:  0.24, x: -3.4, y: -1.7, z: 1.2, qw: 1, qx: -1.57, qy: 1.57, qz: 0},
   'tag_8': {size:  0.24, x: 1.5, y: -1.3, z: 1.2, qw: 1, qx: 0.0, qy: -1.5708, qz: -1.5708},
   'tag_9': {size:  0.24, x: -1.5, y: -2.25, z: 1.3, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 4.7},
   'tag_10': {size:  0.24, x: 1.5, y: 1.55, z: 1.2, qw: 1, qx: 0, qy: -1.5708, qz: -1.5708},
   'tag_11': {size:  0.24, x: 0.55, y: -2.25, z: 1.4, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 4.712},
   'tag_12': {size:  0.24, x: -2.6, y: -2.25, z: 0.8, qw: 1, qx: 1.5708 , qy: 3.141592, qz: 4.7},
   'tag_13': {size:  0.24, x: -1.65, y: 2.7, z: 1.4, qw: 1, qx: 1.57, qy: 0, qz: 1.57},
   'tag_14': {size:  0.24, x: -3.4, y: -1.15, z: 0.8, qx: -1.57, qy: 1.57, qz: 0},
   'tag_15': {size:  0.24, x: -3.4, y: 1.3, z: 0.9, qw: 1, qx: -1.57, qy: 1.57, qz: 0}
  }
